vakt/storage/sql/model.py

```diff
@@ -0,0 +1,109 @@
+import json
+
+from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+from ...policy import Policy
+
+Base = declarative_base()
+
+
+class PolicySubjectModel(Base):
+    """Storage model for policy subjects"""
+
+    __tablename__ = 'policy_subject'
+
+    id = Column(Integer, primary_key=True)
+    uid = Column(String(254), ForeignKey('policy.uid', ondelete='CASCADE'))
+    subject = Column(String(254))
```

Comment:
If we need to use index for subject field we can opt to varchar with more capacity (say, 1000).

Reply:
The implemented approach works fine for String-based (regex-) and rule-based policies. But in order to get full support for vakt regex patterns we need to compile value and store it in the compiled field that later will be used by reverse regex match on the database side. This is needed because vakt regex definitions are some sort of superset of a regex.

Reply:
Agreed with using Text type. 

In regards to regex-based search from the database, I think we should hold off on that for now. In my opinion, we should eventually get rid of the different checkers and just have rule checker. As a user of the package, I would expect the correct checker to be used internally instead of explicitly providing one.

Reply:
Ideally, yes, since rule-based policies supersede the functionality of regex- and string-based ones and thus the latter become not much useful. Let me think about it...